treeherder/client/thclient/client.py

```diff
@@ -681,7 +682,16 @@ def _get_json(self, endpoint, timeout, project=None, **params):
 
         resp = requests.get(uri, timeout=timeout, params=params,
                             headers=self.REQUEST_HEADERS)
-        resp.raise_for_status()
+        try:
+            resp.raise_for_status()
+        except HTTPError as e:
+            response = e.response
+            logger.error("Error submitting data to %s" % response.request.url)
```

Comment:
Ah the URL being included in the HTTPError was just on master at the time of my link above; it's now in requests 2.8.0+.